src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala

```diff
@@ -77,6 +79,7 @@ object ArtifactorySettingsPlugin extends AutoPlugin {
       Seq(
         if (isSnapshot.value) Some(Resolver.mavenLocal) else None,
         Some("Artifactory Release Libs" at artifactoryJvmReleasesResolver.value),
+        Some("Artifactory Release Libs Local" at artifactoryJvmReleasesPublishResolver.value),
```

Comment:
> We add the libs-release-local artifactory repository to the resolvers because even though we publish to it by default, we do not resolved from it, meaning that we can't get the libraries that we just published.

would there be any side effect of doing that? I'm not an sbt expert, maybe @jcasadella may approve with more confidence.

Reply:
The effect is that all Adevinta libraries published in that repository will become available (if you have access of course). I don't see how this can hurt existing projects.